spicy/toolchain/include/ast/types/unit-items/field.h

```diff
@@ -101,7 +101,8 @@ class Field : public hilti::NodeBase, public spicy::trait::isUnitItem {
     Node& itemNode() { return childs()[2]; }
     Node& attributesNode() { return childs()[4]; }
 
-    std::optional<std::pair<Expression, bool>> convertExpression() const;
+    // For unit-level converts, returns the unit type as well.
```

Comment:
Let's add document what this function does, and only then the behavior for unit-level converts, e.g.,

```suggestion
    // Get the `&convert` expression, if any.
    //
    // For unit-level converts, returns the unit type as well.
```